server/index.js

```diff
@@ -75,7 +76,7 @@ if (Config.watchconfig) {
 	FS(configPath).onModify(() => {
 		try {
 			delete require.cache[configPath];
-			global.Config = require('../config/config');
+			global.Config = ConfigLoader.load(require('../config/config'));
```

Comment:
In theory this shouldn't need an argument. `delete require.cache[configPath]` should probably be part of `load`